### ISSUE TITLE
build failed: No such file or directory

### ISSUE THREAD

**Issue by tkamenoko** [nbigaouette/pycors#103]
platform: KDE neon User Edition 5.17(based on ubuntu)
pycors: 0.2.1

Building python failed because of OS Error: No such file or directory

```sh
$ cd ~/develop
$ pycors install =3.8
```

>[3/15] Configure
Error: No such file or directory (os error 2)

but following commands worked as expected.

```sh
$ cd  ~/.pycors/cache/extracted/Python-3.8.0/
$ pycors install =3.8
``` 

IMO, `./configure` should be an absolute path, or run after changing the working directory.

https://github.com/nbigaouette/pycors/blob/a6776da6c7701943f85606664fd124666657e06a/src/commands/install/unix.rs#L125-L131

**Comment by nbigaouette** [comment]
Yes, you got it right. A previous refactor deleted the setting of current working directory, breaking compilation.

A temporary fix is, as you found out, to change to the extracted directory before re-running the process.

I've fixed this in c1ffbfeb2ee846805aa042e84f540af841e22a70 (not yet released). I'll try to publish soon.

**Comment by nbigaouette** [comment]
0.2.2 is building right now which should fix this.
